### PR TITLE
fix(jobs): various bugs

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

- Fix stepper / Title of funnel flow
https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-628
- Fix env variable row showing / hiding secret input
- Fix Romaric small glitch on input show hide passwird
- Showing the good container/git card in overview
- Fixing step summary

image

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - Fix/job various bugs - 2022-12-26T15:50:52Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/9fc3105f-d81e-4018-9e81-58ca55a43dd0/applications)

 Another comment will be posted when deployments are terminated

File: libs/pages/application/src/lib/ui/container-card/container-card.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { ContainerApplicationEntity, JobApplicationEntity } from '@qovery/shared/interfaces'
+import { timeAgo } from '@qovery/shared/utils'
+
+export interface ContainerCardProps {
+  application: ContainerApplicationEntity | JobApplicationEntity
+}
+
+export function ContainerCard(props: ContainerCardProps) {
```
Comment (RemiBonnet) [inline libs/pages/application/src/lib/ui/container-card/container-card.tsx:8]: It's not a card ? just a AboutContentContainer no ?

File: libs/pages/services/src/lib/ui/page-job-create/step-summary/step-summary.tsx [hunk 2]
```diff
@@ -25,7 +25,9 @@ export function StepSummary(props: StepSummaryProps) {
     <div>
       <div className="mb-10">
         <div className="flex justify-between mb-2 items-center">
-          <h3 className="text-text-700 text-lg">Ready to create your Cronjob</h3>
+          <h3 className="text-text-700 text-lg">
+            Ready to create your {props.jobType === ServiceTypeEnum.CRON_JOB ? 'Cron' : 'Lifecycle'} job
```
Comment (RemiBonnet) [inline libs/pages/services/src/lib/ui/page-job-create/step-summary/step-summary.tsx:29]: You can use `isCronJob` here

**Review by RemiBonnet — COMMENTED** [review]
Well done @bdebon I have two suggestions to improve it !

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z9fc3105f-z2400abca-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z9fc3105f-z3de41a92-gtw.za8ad0659.bool.sh)

**Pull request merged**